assets/images/bankicons/discover.svg

```diff
@@ -0,0 +1,13198 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Woah wtf this svg is 13000 lines!?

Reply:
<img width="460" alt="Screenshot 2021-08-18 at 1 11 13 AM" src="https://user-images.githubusercontent.com/85645967/129789987-ae49c27b-3e4d-41a8-9c12-b983dfc68628.png">

That's the bigger file

Reply:
Asked in the thread where we generated these why that one is gigantic